gcm/README.md

```diff
@@ -0,0 +1,98 @@
+# Parse SDK Android GCM
+GCM support for Parse Android apps
+
+## Deprecated
+Please note that GCM is deprecated in favor of FCM. This module exists as a backwards compatible solution for projects already using GCM. New apps should instead use FCM.
+
+## Setup
+
+### Installation
+
+Add dependency to the application level `build.gradle` file.
+
+```groovy
+dependencies {
+    implementation 'com.parse:parse-android:latest.version.here'
+    implementation 'com.parse:parse-android-gcm:latest.version.here'
+
+    implementation "com.google.android.gms:play-services-gcm:latest.version.here"
```

Comment:
this shouldn't be necessary given you declared play-services-gcm as an api dependency